config/samples/flows_v1beta1_flowcollector.yaml

```diff
@@ -8,23 +8,26 @@ spec:
   agent:
     type: EBPF
     ebpf:
-      imagePullPolicy: IfNotPresent
+      imagePullPolicy: Always
       sampling: 50
       cacheActiveTimeout: 5s
       cacheMaxFlows: 100000
-      interfaces: [ ]
-      excludeInterfaces: [ "lo" ]
-      logLevel: info
+      interfaces: [ "eth0", "veth5871bad5" ]
+      excludeInterfaces: [ ]
+      logLevel: debug
+      privileged: true      
```

Comment:
@astoycos the privileged mode here is not something that will be required when running with bpfd, or is it? It would increase the agent privileges requirements (while I guess they should be lowered). Or is it just something you did for testing/debugging convenience but that will not be required ultimately?

Reply:
> Or is it just something you did for testing/debugging convenience but that will not be required ultimately?

Yep :)